src/readii/negative_controls.py

```diff
@@ -137,13 +152,16 @@ def makeRandomRoi(
     # Delete the input image to save space
     del baseImage
 
+    # Set the random seed for np random generator
+    randNumGen = np.random.default_rng(seed=randomSeed)
+
     # Now iterate over the pixels of the ROI in the image and randomly generate a new value for them
     for x in range(baseROISize[0]):
         for y in range(baseROISize[1]):
             for z in range(baseROISize[2]):
                 if baseROI.GetPixel(x, y, z) == roiLabel:
                     # Randomly assigning the current value to the range [maxVoxelVal, maxVoxelVal]
-                    mapped_value = random.randint(minVoxelVal, maxVoxelVal)
+                    mapped_value = int(randNumGen.integers(low=minVoxelVal, high=maxVoxelVal, endpoint=True))
```

Comment:
not a major issue but it looks like `randNumGen.integers` requires `low` and `high` to be "int-like", but `maxVoxelVal` & `minVoxelVal` are floats by default. 

do you think theres any issue if we cast both of them to `int()` to satisfy the parameter type?